share/ci/scripts/linux/install_boost.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Why is install_boost.sh needed at all? Is there a reason why the ASWF containers' version of boost is inadequate?

Reply:
I wound up copying the entire share/ci/scripts directory from OpenEXR. This can be removed along with a few other scripts probably if that makes things cleaner.